sql/_hny_join.sql

```diff
@@ -0,0 +1,196 @@
+        extremely_low_income_units numeric,
+        very_low_income_units numeric,
+        low_income_units numeric,
+        moderate_income_units numeric,
+        middle_income_units numeric,
+        other_income_units numeric,
+        studio_units numeric,
+        "1_br_units" numeric,
+        "2_br_units" numeric,
+        "3_br_units" numeric,
+        "4_br_units" numeric,
+        "5_br_units" numeric,
+        "6_br+_units" numeric,
+        unknown_br_units numeric,
+        counted_rental_units numeric,
+        counted_homeownership_units numeric
+    )
+
+    hny_geocode_results (
+        * uid,
+        geo_bin,
+        geo_bbl,
+        geo_latitude,
+        geo_longitude
+
+    )
+
+    
+OUTPUTS: 
+    HNY_devdb(
+        hny_id text,
+        job_number text,
+        classa_hnyaff text,
+        all_hny_units text,
+        one_dev_to_many_hny integer,
+        one_hny_to_many_dev integer,
+        project_start_date text,
+        project_completion_date text,
+        extremely_low_income_units numeric,
+        very_low_income_units numeric,
+        low_income_units numeric,
+        moderate_income_units numeric,
+        middle_income_units numeric,
+        other_income_units numeric,
+        studio_units numeric,
+        "1_br_units" numeric,
+        "2_br_units" numeric,
+        "3_br_units" numeric,
+        "4_br_units" numeric,
+        "5_br_units" numeric,
+        "6_br+_units" numeric,
+        unknown_br_units numeric,
+        counted_rental_units numeric,
+        counted_homeownership_units numeric
+    )
+*/
+
+-- 5) Identify relationships between devdb records and hny records
+DROP TABLE IF EXISTS HNY_devdb;
+WITH 
+	-- Find cases of many-hny-to-one-devdb, after having filtered to highest priority
+	many_developments AS (SELECT hny_id
+				FROM HNY_matches
+				GROUP BY hny_id
+                HAVING COUNT(*)>1),
+				
+	-- Find cases of many-devdb-to-one-hny, after having filtered to highest priority
+	many_hny AS (SELECT a.job_number
+				FROM HNY_matches a
+				GROUP BY a.job_number
+                HAVING COUNT(*)>1),	
+
+	-- Add relationship flags, where '1' in both flags means a many-to-many relationship
+    RELATEFLAGS_hny_matches AS
+    (SELECT m.*,
+		(CASE 
+			WHEN hny_id IN (SELECT DISTINCT hny_id FROM many_developments) THEN 1
+			ELSE 0 
+		END) AS one_hny_to_many_dev,
+		(CASE 
+			WHEN job_number IN (SELECT DISTINCT job_number FROM many_hny) THEN 1
+			ELSE 0
+		END) AS one_dev_to_many_hny
+    FROM HNY_matches m), --- maybe this is where the hny_geo could be brought in again
+
+-- 5) ASSIGN MATCHES   
+	-- a) For one dev to many hny, group by job_number and sum unit fields
+	one_to_many AS (SELECT 
+        string_agg(r.hny_id, ', ') AS hny_id,
+        r.job_number, 
+        SUM(COALESCE(r.all_counted_units::int, '0'))::text AS classa_hnyaff,
+        SUM(COALESCE(r.total_units::int, '0'))::text AS all_hny_units,
+        r.one_dev_to_many_hny,
```

Comment:
I think these columns are well named, make the output much readable. If this is important info for housing do we want to move them even further left to come right after the job number? Worry that they may be hard to find where they are. Don't know what housing expects though

Reply:
Thoughts on moving the `one_xxxxxx` columns right after job number and shifting the `class_hynaff `and `all_hny_units` after `project_completion_date` so that the columns related to units are bundled together?